FirebaseRemoteConfig/Sources/RCNConfigDBManager.m

```diff
@@ -861,12 +909,44 @@ - (void)loadExperimentWithCompletionHandler:(RCNDBCompletion)handler {
 
   NSArray *params = @[ key ];
   [self bindStringsToStatement:statement stringArray:params];
-  NSData *experimentData;
+  NSMutableArray *results = [self loadValuesFromStatement:statement];
+  return results;
+}
+
+- (NSArray *)loadRolloutTableFromKey:(NSString *)key {
+  RCN_MUST_NOT_BE_MAIN_THREAD();
+  const char *SQL = "SELECT value FROM " RCNTableNameRollout " WHERE key = ?";
+  sqlite3_stmt *statement = [self prepareSQL:SQL];
+  if (!statement) {
+    return nil;
+  }
+  NSArray *params = @[ key ];
+  [self bindStringsToStatement:statement stringArray:params];
+  NSMutableArray *results = [self loadValuesFromStatement:statement];
+  // There should be only one entry in this table.
+  if (results.count != 1) {
+    return nil;
+  }
+  NSArray *rollout;
+  // Convert from NSData to NSArray
+  if (results[0]) {
+    NSError *error;
+    rollout = [NSJSONSerialization JSONObjectWithData:results[0] options:0 error:&error];
```

Comment:
Can we have a debug log if error happens here? Otherwise I think we can just pass by nil.

Reply:
Added.